src/custom/hooks/useSwapCallback.ts

```diff
@@ -29,6 +29,26 @@ function calculateValidTo(deadline: number): number {
   return Math.min(validTo, MAX_VALID_TO_EPOCH)
 }
 
+const _computeInputAmountForSignature = (params: {
+  input: CurrencyAmount
+  inputWithSlippage: CurrencyAmount
+  fee?: CurrencyAmount
+  kind: OrderKind
+}) => {
+  const { input, inputWithSlippage, fee, kind } = params
+  // When POSTing the order, we need to check inputAmount value depending on trade type
+  // If we don't have an applicable fee amt, return the inputAmount adjusted for fee
```

Comment:
From how it's implemented it looks like this comment is outdated.

```suggestion
  // If we don't have an applicable fee amt, return the input as is
```